lib/improver/psychrometric_calculations/psychrometric_calculations.py

```diff
@@ -0,0 +1,353 @@
+        pass
+
+    def __repr__(self):
+        """Represent the configured plugin instance as a string."""
+        result = ('<Utilities>')
+        return result
+
+    @staticmethod
+    def specific_heat_of_moist_air(mixing_ratio):
+        """
+        Calculate the specific heat capacity for moist air by combining that of
+        dry air and water vapour in proportion given by the specific humidity.
+
+        Args:
+            mixing_ratio : iris.cube.Cube
+                Cube of specific humidity (fractional).
+        Returns:
+            iris.cube.Cube
+                Specific heat capacity of moist air (J kg-1 K-1).
+        """
+        specific_heat = ((-1.*mixing_ratio + 1.) * cc.U_CP_DRY_AIR +
+                         mixing_ratio * cc.U_CP_WATER_VAPOUR)
+        specific_heat.rename('specific_heat')
+        return specific_heat
+
+    @staticmethod
+    def latent_heat_of_condensation(temperature_input):
+        """
+        Calculate a temperature adjusted latent heat of condensation for water
+        vapour using the relationship employed by the UM.
+
+        Args:
+            temperature : iris.cube.Cube
+                A cube of air temperatures (Celsius, converted if not).
+        Returns:
+            iris.cube.Cube
+                Temperature adjusted latent heat of condesation (J kg-1).
+        """
+        temperature = temperature_input.copy()
+        temperature.convert_units('celsius')
+        latent_heat = (-1. * cc.U_LATENT_HEAT_T_DEPENDENCE * temperature +
+                       cc.U_LH_CONDENSATION_WATER)
+        latent_heat.units = cc.U_LH_CONDENSATION_WATER.units
+        latent_heat.rename('latent_heat_of_condensation')
+        return latent_heat
+
+    @staticmethod
+    def calculate_enthalpy(mixing_ratio, specific_heat, latent_heat,
+                           temperature):
+        """
+        Calculate the enthalpy (total energy per unit mass) of air (J kg-1).
+
+        Args:
+            mixing_ratio : iris.cube.Cube
+                Cube of mixing ratios.
+            specific_heat : iris.cube.Cube
+                Cube of specific heat capacities of moist air (J kg-1 K-1).
+            latent_heat : iris.cube.Cube
+                Cube of latent heats of condensation of water vapour
+                (J kg-1).
+            temperature : iris.cube.Cube
+                A cube of air temperatures (K).
+        Returns:
+           enthalpy : iris.cube.Cube
+               A cube of enthalpy values calculated at the same points as the
+               input cubes (J kg-1).
+        """
+        enthalpy = latent_heat * mixing_ratio + specific_heat * temperature
+        enthalpy.rename('enthalpy_of_air')
+        return enthalpy
+
+    @staticmethod
+    def calculate_d_enthalpy_dt(mixing_ratio, specific_heat,
+                                latent_heat, wet_bulb_temperature):
+        """
+        Calculate the enthalpy gradient with respect to temperature.
+
+        Args:
+
```

Comment:
Arguments need to be added to this function.

Reply:
:white_check_mark: